pkg/server/http_status.go

```diff
@@ -333,6 +334,7 @@ func (s *Server) startHTTPServer() {
 	router.HandleFunc("/debug/pprof/trace", pprof.Trace)
 	// Other /debug/pprof paths not covered above are redirected to pprof.Index.
 	router.PathPrefix("/debug/pprof/").HandlerFunc(pprof.Index)
+	router.HandleFunc("/debug/traceevent", traceeventHandler)
```

Comment:
Do we still need the http api if the `json` session variable is used?

Reply:
The benefit of keeping the json here, is that we don't need to handle the dump trigger config twice:
once in SQL and then http ... this would bring a lot conveniency

They're for different scenario: sink to log is mainly used for transaction bug reproduce and diagnosis
sink to http endpoint always used together with http://ui.prefetto.dev and is mainly for performance diagnosis